app/src/main/java/org/standardnotes/notes/SettingsActivity.kt

```diff
@@ -35,6 +38,24 @@ class SettingsActivity : BaseActivity() {
         return super.onOptionsItemSelected(item)
     }
 
+    override fun onDestroy() {
+        super.onDestroy()
+        // TODO does this make sense here?
+        ExportUtil.clearExports(this)
```

Comment:
Yeah, not sure when exactly we should clear the exported file out, but it does stick around in the app's storage.